packages/ember-glimmer/lib/environment.js

```diff
@@ -224,12 +227,15 @@ export default class Environment extends GlimmerEnvironment {
     return false;
   }
 
-  getComponentDefinition(path) {
+  getComponentDefinition(path, parentMeta = {}) {
     let name = path[0];
     let definition = this._components[name];
```

Comment:
The name needs to be the resolved name (not the local name) before looking up the cached compiled definition. We need to refactor to address this, maybe use the _debugContainerKey or less debug friendly, use guidFor(ComponentClass) as the definition key.